### PR TITLE
Add shortcut to open template for current context

### PR DESCRIPTION
**Pull request by tokejepsen**
## Changelog Description
Port of https://github.com/ynput/OpenPype/pull/6181

### CODE REVIEW

**Comment by tokejepsen** [comment]
@iLLiCiTiT it looks like https://github.com/ynput/OpenPype/blob/develop/openpype/widgets/message_window.py has been removed?

**Comment by iLLiCiTiT** [comment]
> @iLLiCiTiT it looks like https://github.com/ynput/OpenPype/blob/develop/openpype/widgets/message_window.py has been removed?

It's been moved to `ayon_core.tools.utils.dialogs`.

File: client/ayon_core/hosts/maya/api/menu.py [hunk 1]
```diff
@@ -21,8 +22,10 @@
     create_placeholder,
     update_placeholder,
     build_workfile_template,
-    update_workfile_template,
+    update_workfile_template
 )
+from openpype.tools.workfile_template_build import open_template_ui
```
Comment (iLLiCiTiT) [inline client/ayon_core/hosts/maya/api/menu.py:27]:
```suggestion
from ayon_core.tools.workfile_template_build import open_template_ui
```

**Review by iLLiCiTiT — APPROVED** [review]
Code changes make sense.

**Review by jakubjezek001 — APPROVED** [review]
works in Nuke

**Pull request merged**